db/seeds.rb

```diff
@@ -0,0 +1,19 @@
+module Shop
```

Comment:
I am not sure about this file. `db/seeds.rb` file should populate database with sample data. Here you have module with "database" definition. Those constants should stay where they were. Here you should just call some "add" methods with sample data.